test/functional/feature_block.py

```diff
@@ -1263,6 +1263,10 @@ def run_test(self):
         b89a = self.update_block("89a", [tx])
         self.send_blocks([b89a], success=False, reject_reason='bad-txns-inputs-missingorspent', reconnect=True)
 
+        # Don't use v2transport for the remaining tests, the large reorg is very slow with the unoptimized python ChCha20 implementation
```

Comment:
d065b33: maybe say only for `LARGE_REORG_SIZE` tests? because the last 2 tests have v2 peers after [reconnection](https://github.com/bitcoin/bitcoin/blob/d8165db8e889f423351ae35fd375c982e793d136/test/functional/feature_block.py#L1312-L1320) due to block height mismatch/invalid block header version.